### PR TITLE
finished calendar task

### PR DESCRIPTION
**Pull request by tiserett**
- [DEMO LINK](https://tiserett.github.io/layout_calendar/)
- [TEST REPORT LINK](https://tiserett.github.io/layout_calendar/report/html_report/)

```
Implement start-day modifier for the calendar with mon, tue, wed, thu, fri, sat and sun values
Use [@each](https://sass-lang.com/documentation/at-rules/control/each) to create all the modifiers
The month should start at the correct column (Monday is the 1st, Friday is the 5th)
You can just add correct margin-left for the first day
```
I haven't finished this part of task because I don't really understand what am I supposed to do

### CODE REVIEW

**Review by adrqq — CHANGES_REQUESTED** [review]
You should add $days variable with name and key,  for example, Monday needs to be first so it doesn't need left distance:
$days:
  "mon" 0
  and Tuesday needs to be second so you writte:
  $days:
   "mon" 0
   "tue" your distance from left
   and so on 
   
Then you need to add @each with will add a different style to your first block depending on the day that you wrote in class 
  @each $start, $day-1-possition in $days {
     &__day--start-#{$start} {                           // $start is the name of your day 
      margin-left: $day-1-possition;       // $day-1-possition is a key that you added in $days 
     }
  }
  
  Calculate your distance using variables

**Review by adrqq — APPROVED** [review]
good job)

File: src/index.html [hunk 1]
```diff
@@ -7,6 +7,40 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <section class="section">
+      <div class="calendar calendar--month-length-31 calendar__start-day--sun">
+        <div class="calendar__day calendar__day--1"></div>
+        <div class="calendar__day calendar__day--2"></div>
```
Comment (AlinaMatuschak) [inline src/index.html:13]: don't need to create 1, 2 ... modifiers. Use nth-child

File: src/styles/blocks/calendar.scss [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    @for $index from 1 through 31 {
+      &--#{$index}::before {
+        content: "#{$index}";
+      }
+    }
+
+    &:hover {
+      transform: translateY(-20px);
+      background-color: pink;
+      cursor: pointer;
```
Comment (AlinaMatuschak) [inline src/styles/blocks/calendar.scss:44]:
```suggestion
    transition: 0.5s;
    cursor: pointer;

    @for $index from 1 through 31 {
      &--#{$index}::before {
        content: "#{$index}";
      }
    }

    &:hover {
      transform: translateY(-20px);
      background-color: pink;
```

**Review by AlinaMatuschak — APPROVED** [review]
🚀

**Pull request closed without merging**